lib/core/kotekanTrackers.cpp

```diff
@@ -0,0 +1,68 @@
+#include "kotekanTrackers.hpp"
+
+#include "kotekanLogging.hpp" // for ERROR_NON_OO
+
+#include "fmt.hpp"  // for format, fmt
+#include "json.hpp" // for json, basic_json<>::value_type
+
+#include <functional> // for _Bind_helper<>::type, _Placeholder, bind, _1, placeholders
+#include <stdexcept>  // for runtime_error
+#include <utility>    // for pair
+
+namespace kotekan {
+namespace trackers {
+
+KotekanTrackers::KotekanTrackers() {}
+
+KotekanTrackers::~KotekanTrackers() {
+    restServer::instance().remove_get_callback("/trackers");
+}
+
+KotekanTrackers& KotekanTrackers::instance() {
+    static KotekanTrackers _instance;
+    return _instance;
+}
+
+void KotekanTrackers::register_with_server(restServer* rest_server) {
+    using namespace std::placeholders;
+    rest_server->register_get_callback("/trackers",
```

Comment:
I would like to have two end points here.  `/trackers_current` and `/trackers`.   This current endpoint can stay the same, but we should add a `trackers_current` which returns just the current min/max/avg/std/etc. for each tracker.   I can see the JSON returned from `/trackers` getting very large, and JSON processing in Javascript is somewhat slow, so for realtime watching of stats we really just want the current values.